packages/sui-studio/src/components/demo/try-require.js

```diff
@@ -43,6 +43,7 @@ const tryRequire = ({category, component}) => {
         const bundler = reqComponentsPlayGround(`./${category}/${component}/playground`)
         bundler(playground => resolve(playground))
       } catch (e) {
+        const Component = exports.default
         return resolve(`return (<${Component.displayName || Component.name} />)`)
```

Comment:
this isn't working and it wasn't working as exports/Component is undefined at this point. Is it ok if I change it to a console.error instead? Looks like this is executed if the playground can't be loaded